### PR TITLE
Add ability to deploy a test server and test the upgrade

### PR DESCRIPTION
**Pull request by cpanato**
Next stage is to refactor to use terraform. this is a quick and small win for now

### CODE REVIEW

File: server/spinmint.go [hunk 1]
```diff
@@ -173,6 +173,61 @@ func waitForBuildAndSetupSpinmint(pr *model.PullRequest) {
 	commentOnIssue(pr.RepoOwner, pr.RepoName, pr.Number, message)
 }
 
+func waitForBuildAndSetupSpinmintUpgrade(pr *model.PullRequest) {
```
Comment (crspeller) [inline server/spinmint.go]: Can we reuse this code instead of copy paste and just parameterize this function?

Comment (cpanato) [reply server/spinmint.go]: yep, will change

Comment (cpanato) [reply server/spinmint.go]: done

**Comment by crspeller** [comment]
@cpanato Merge when ready.

**Pull request merged**